src/env.h

```diff
@@ -1292,6 +1292,7 @@ class Environment : public MemoryRetainer {
   bool emit_err_name_warning_ = true;
   size_t async_callback_scope_depth_ = 0;
   std::vector<double> destroy_async_id_list_;
+  int stack_trace_limit_ = 10;
```

Comment:
Since this is effectively a constant we usually would name this like kStackTraceLimit